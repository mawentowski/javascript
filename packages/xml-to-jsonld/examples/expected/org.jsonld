{
  "@context": "https://schema.org",
  "@type": "Organization",
  "name": "Acme Inc.",
  "url": "https://acme.example",
  "logo": "https://acme.example/logo.png",
  "sameAs": [
    "https://twitter.com/acme"
  ],
  "contactPoint": [
    {
      "@type": "ContactPoint",
      "telephone": "+1-555-0100",
      "contactType": "customer support"
    }
  ]
}