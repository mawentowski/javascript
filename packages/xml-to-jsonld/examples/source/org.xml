<!-- <?xml-model href="../packages/schemas/rnc/site-seo.rnc"
            type="application/relax-ng-compact-syntax"?> -->

<OrgRoot>
  <name>Acme Inc.</name>
  <url>https://acme.example</url>
  <logo>
    <ImageObject>
      <url>https://acme.example/logo.png</url>
    </ImageObject>
  </logo>
  <sameAs>https://twitter.com/acme</sameAs>
  <contactPoint>
    <ContactPoint>
      <telephone>+1-555-0100</telephone>
      <contactType>customer support</contactType>
    </ContactPoint>
  </contactPoint>
</OrgRoot>