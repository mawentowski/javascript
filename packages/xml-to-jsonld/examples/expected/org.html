<h1>Acme Inc.</h1><p><a href="https://acme.example">https://acme.example</a></p>